cvat-ui/src/components/cvat-app.tsx

```diff
@@ -149,7 +149,9 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
         });
 
         core.logger.configure(() => window.document.hasFocus, userActivityCallback);
-        EventRecorder.initSave();
+        if (user) {
+            EventRecorder.initSave();
+        }
```

Comment:
I am not sure this condition may be true in componentDidMount, because it first mounted and then authorized method is called to receive current user